worker/src/main/scala/org/apache/celeborn/service/deploy/worker/Worker.scala

```diff
@@ -337,7 +337,7 @@ private[celeborn] class Worker(
       conf.workerCleanThreads)
   val asyncReplyPool: ScheduledExecutorService =
     ThreadUtils.newDaemonSingleThreadScheduledExecutor("worker-rpc-async-replier")
-  val timer = new HashedWheelTimer()
+  val timer = new HashedWheelTimer(ThreadUtils.namedThreadFactory("worker-timer"))
```

Comment:
should we use `namedSingleThreadFactory` instead?

```suggestion
  val timer = new HashedWheelTimer(ThreadUtils.namedSingleThreadFactory("worker-timer"))
```

Reply:
HashedWheelTimer will generate only one thread. So I think this suggestion is reasonable.